clients/curl-client.rst

```diff
@@ -12,17 +12,16 @@ To install the cURL client, run:
 
     $ composer require php-http/curl-client
 
-Usage
------
-
-The cURL client needs a :ref:`message <message-factory>` and a :ref:`stream <message-factory>`
-factory in order to to work::
+.. include:: includes/install-message-factory.inc
 
-    use Http\Client\Curl\Client;
+.. include:: includes/install-discovery.inc
```

Comment:
@sagikazarmark is this what you meant by mentioning the factories in the installation instructions?